modules/salt/pages/yomi.adoc

```diff
@@ -0,0 +1,174 @@
+[[yomi.installer]]
+= Install with Yomi
+
+Yomi (yet one more installer) is an installer for openSUSE operating systems.
+It is a SaltStack state, and can be used for installing Salt clients.
```

Comment:
Yomi is for installing openSUSE, SLE or MicroOS operating systems (similar to autoyast), not for installing Salt.